### PR TITLE
Bevy 0.13

### PR DESCRIPTION
**Pull request by Lethja**
Updated code to build for Bevy 0.13

# PR Status
- [X] Builds the crate successfully (`cargo test`)
- [X] Build doesn't emit any warnings
- [X] Bevy 0.13.1 has been released
- [X] Example code works as intended (when combined with #65)

### CODE REVIEW

**Comment by Lethja** [comment]
Example scenes will panic when clicking any select-able object in the scene with the following message on commit 8c31bea:
```rust
thread 'main' panicked at ~/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wgpu-0.19.3/src/backend/wgpu_core.rs:3006:5:
wgpu error: Validation Error

Caused by:
    In Device::create_render_pipeline
      note: label = `opaque_mesh_pipeline`
    Error matching ShaderStages(FRAGMENT) shader requirements against the pipeline
    Shader global ResourceBinding { group: 1, binding: 0 } is not available in the pipeline layout
    Storage class Storage { access: StorageAccess(LOAD) } doesn't match the shader Uniform

```

**Comment by Lethja** [comment]
> Example scenes will panic when clicking any select-able object in the scene with the following message on commit [8c31bea](https://github.com/ForesightMiningSoftwareCorporation/bevy_transform_gizmo/commit/8c31beab11f2c2615696a51bd69457fde186ac8e):

Issue will be resolved by Bevy 0.13.1 see https://github.com/bevyengine/bevy/pull/12032

**Comment by zachbateman** [comment]
Hey @Lethja, the PR I just submitted (#65) should take care of that wgpu panic.

File: src/mesh/mod.rs [hunk 1]
```diff
@@ -30,24 +28,18 @@ pub fn build_gizmo(
...
     }));
-    let plane_mesh = meshes.add(Mesh::from(shape::Plane::from_size(plane_size)));
-    let sphere_mesh = meshes.add(
-        Mesh::try_from(shape::Icosphere {
-            radius: 0.2,
-            subdivisions: 3,
-        })
-        .unwrap(),
-    );
+    let plane_mesh = meshes.add(Mesh::from(Plane3d::default()));
```
Comment (zachbateman) [inline src/mesh/mod.rs:41]: The new default plane is too big for the gizmo.  The below modified line should fix this:

`let plane_mesh = meshes.add(Mesh::from(Plane3d::default().mesh().size(plane_size, plane_size)));`

Comment (Lethja) [reply src/mesh/mod.rs]: Updated in ccf615d

**Comment by Lethja** [comment]
Ready for another CI attempt at 0aa13f8

**Comment by aevyrie** [comment]
Looks great, thank you!

**Pull request merged**